modules/cli/src/main/scala/scala/cli/commands/bsp/Bsp.scala

```diff
@@ -29,6 +30,14 @@ object Bsp extends ScalaCommand[BspOptions] {
         val content = os.read.bytes(os.Path(optionsPath, os.pwd))
         readFromArray(content)(SharedOptions.jsonCodec)
       }.getOrElse(options.shared)
+  private def latestLauncherOptions(options: BspOptions): LauncherOptions =
+    options.jsonLauncherOptions
+      .map(path => os.Path(path, os.pwd))
+      .filter(path => os.exists(path) && os.isFile(path))
+      .map { optionsPath =>
```

Comment:
```suggestion
      .collect { case optionsPath if os.exists(optionsPath) && os.isFile(optionsPath)) =>
```
super nitpick

Reply:
will address separately, got no time to restart the CI on this 👍